torchrl/collectors/distributed/utils.py

```diff
@@ -0,0 +1,150 @@
+import subprocess
+import time
+
+from torchrl.collectors.distributed.default_configs import (
+    DEFAULT_SLURM_CONF,
+    DEFAULT_SLURM_CONF_MAIN,
+    TCP_PORT,
+)
+from torchrl.collectors.distributed.generic import _distributed_init_delayed
+from torchrl.collectors.distributed.rpc import _rpc_init_collection_node
```

Comment:
Consider making `_distributed_init_delayed` and `_rpc_init_collection_node` public